### DISCUSSION TITLE
Some problems with the emulator:

### DISCUSSION THREAD

**Discussion by MM2WOOF** [rafaelvcaetano/melonDS-android#90] [General]
I like MelonDS Android, it really looks quite promising, even so I will say some drawbacks that I have with said emulator:
- The emulator does not have a Menu option, I would like it to be done to avoid moving the bottom side of the cell phone screen, which is uncomfortable
- You cannot change the screen position or the size of the Nintendo DS windows and buttons, I would like to see that option in the future
- The firmware of the DSi has many problems, if you can play Nintendo DS games there but it has errors such as saving and loading status, lack of compatibility with games like Pokemon B / W, among others.

That's all I can say, I'm not saying that the emulator is bad but I must mention the bugs it has, I wish the people of MelonDS luck, a greeting.

**Comment by rafaelvcaetano** [comment]
Thanks for your feedback!

> The emulator does not have a Menu option, I would like it to be done to avoid moving the bottom side of the cell phone screen, which is uncomfortable

Thanks for the suggestions. I will consider this for a future release.

> You cannot change the screen position or the size of the Nintendo DS windows and buttons, I would like to see that option in the future

That feature is in development.

> The firmware of the DSi has many problems, if you can play Nintendo DS games there but it has errors such as saving and loading status, lack of compatibility with games like Pokemon B / W, among others.

The DSi firmware is experimental and it is known that there are serious problems with it. If you believe you have a concrete issue, I would recommend reporting it to the developers of the PC version.
